### ISSUE TITLE
Trying to run vr mod could not switch resolution

### ISSUE THREAD

**Issue by lordKushnu** [cybensis/OutwardVR#30]
Keep getting this error when launching the game
Screen: DX11 could not switch resolution (1920x1080 fs=1 hz=0)

Are there any solutions?

**Comment by Patola** [comment]
Yeah, I am trying to run this mod on Linux and I have the same issue. Even tried different values for the `Outward_Defed/OptionSettings.oos` configuration file but it seems it tries to change resolution but it's unable to do so. Tried fullscreen and windowed, 0x0, -1x-1, removing the file altogether, the resolution of my quest3 headset (4128x2208 or 2064×2208 per eye) but nothing seems to work.

**Comment by Patola** [comment]
Example of one of the failed dialogs when I try to use 2064x2208 (strangely enough, it doesn't seem to have used the Resolution_Y of 2208 that I configured for it, instead trying 1440).
![er](https://github.com/user-attachments/assets/781055f0-dc8f-4ca2-a16c-fedc44e24d69)

**Comment by Patola** [comment]
Interesting is that if I put the resolution of my monitor and Windowed 0 (hence, fullscreen), it first tries exclusive fullscreen, then fails, then fails in Windowed mode. I wonder if it's trying to set this up in the headset or the screen.
![er2](https://github.com/user-attachments/assets/664ef8f6-29e9-465f-9b2e-3718b065b4f8)

**Comment by Kuraika** [comment]
> Interesting is that if I put the resolution of my monitor and Windowed 0 (hence, fullscreen), it first tries exclusive fullscreen, then fails, then fails in Windowed mode. I wonder if it's trying to set this up in the headset or the screen. ![er2](https://github.com/user-attachments/assets/664ef8f6-29e9-465f-9b2e-3718b065b4f8)

I get the very same error message.

**Comment by duke54** [comment]
I have the same problem :(

**Comment by duke54** [comment]
I have the same problem :(

However, the game can still be played in VR using UUVR through the 'Rai Pal' launcher: https://github.com/raicuparta/rai-pal?tab=readme-ov-file

**Comment by lordKushnu** [comment]
The mod is for the previous version of the game, people have had success using Depot downloader to downgrade first before installing the mod

**Comment by duke54** [comment]
> The mod is for the previous version of the game, people have had success using Depot downloader to downgrade first before installing the mod

Do you have any details as to the manifest number that works? I downloaded the two most recent manifests for the mono branch, but the VR wouldn't even start at all for either one.

**Comment by Patola** [comment]
> I have the same problem :(
> 
> However, the game can still be played in VR using UUVR through the 'Rai Pal' launcher: https://github.com/raicuparta/rai-pal?tab=readme-ov-file

Is it as good as this mod, though? Does it allow one to use melée weapons with the spatial controller?

**Comment by Fstash** [comment]
I get the same error, cannot get this VR mod to work like everyone else, too bad i hear this game is worth trying only to buy it and find out this VR mod is broken, stuck here like everyone else. Hopefully this gets repaired one day.

**Error
Couldn't switch to resolution 3840x2160 failed
Screen DX11 could not switch resolution (3840x2160 fs=1 hz=0)**
